views/_includes/layouts/base.html

```diff
@@ -19,8 +20,22 @@
                         <img src="/images/KainosLogo.png" width="150" height="50" alt="Kainos Logo">
                     </a>
                     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
-                        <div class="navbar-nav">
-                            <a class="nav-item nav-link active" href="/">Home<span class="sr-only">(current)</span></a>
+                        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Link 1</a>
+                            </li>
+                            <!-- Each item to be added list item here -->
+                        </ul>
+                        <div class="right-aligned">
```

Comment:
This causes an accessibility issue, as if CSS is turned off nothing appears here